src/vs/vscode.proposed.d.ts

```diff
@@ -1141,4 +1141,35 @@ declare module 'vscode' {
 		alwaysShow?: boolean;
 	}
 	//#endregion
+
+	//#region Tree Item Label Highlights
+	/**
+	 * Label describing the [Tree item](#TreeItem)
+	 */
+	export interface TreeItemLabel {
+
+		/**
+		 * A human-readable string describing the [Tree item](#TreeItem).
+		 */
+		label: string;
+
+		/**
+		 * Ranges in the label to highlight.
+		 */
+		highlights?: [number][number][];
```

Comment:
```suggestion
		highlights?: [number,number][];
```

Reply:
I think like that. Not even sure what `[number][number][]` means?

Reply:
I also do not know 😄 . It was a typo.